expected/13.12/extra/encodings.out

```diff
@@ -34,6 +34,41 @@
 -- WIN1256
 -- WIN1257
 -- WIN1258, not tested
+-- ================
+-- check all data in UTF8
+-- ================
+CREATE EXTENSION sqlite_fdw;
+CREATE SERVER sqlite_svr FOREIGN DATA WRAPPER sqlite_fdw
+OPTIONS (database '/tmp/sqlitefdw_test.db');
+CREATE FOREIGN TABLE "Unicode data"(i text OPTIONS (key 'true'), t text) SERVER sqlite_svr;
+SELECT * FROM "Unicode data";
+  i  |                                                           t                                                            
+-----+------------------------------------------------------------------------------------------------------------------------
+ jap | いろはにほへと ちりぬるを わかよたれそ つねならむ うゐのおくやま けふこえて あさきゆめみし ゑひもせす.
+ bul | Ах, чудна българска земьо, полюшвай цъфтящи жита.
+ rus | Широкая электрификация южных губерний даст мощный толчок подъёму сельского хозяйства.
+ aze | Zəfər, jaketini də, papağını da götür, bu axşam hava çox soyuq olacaq.
+ arm | Բել դղյակի ձախ ժամն օֆ ազգությանը ցպահանջ չճշտած վնաս էր եւ փառք։
+ ukr | Гей, хлопці, не вспію — на ґанку ваша файна їжа знищується бурундучком.
+ eus | Permin gox dabiltzu yoskiñ.
+ bel | У рудога вераб’я ў сховішчы пад фатэлем ляжаць нейкія гаючыя зёлкі.
+ gre | Τάχιστη αλώπηξ βαφής ψημένη γη, δρασκελίζει υπέρ νωθρού κυνός
+ gle | Chuaigh bé mhórshách le dlúthspád fíorfhinn trí hata mo dhea-phorcáin bhig.
+ spa | Quiere la boca exhausta vid, kiwi, piña y fugaz jamón.
+ kor | 키스의 고유조건은 입술끼리 만나야 하고 특별한 기술은 필요치 않다.
+ lav | Ķieģeļu cepējs Edgars Buls fraku un hūti žāvē uz čīkstošām eņģēm.
+ pol | Pchnąć w tę łódź jeża lub ośm skrzyń fig.
+ fra | Dès Noël où un zéphyr haï me vêt de glaçons würmiens je dîne d’exquis rôtis de bœuf au kir à l’aÿ d’âge mûr & cætera !
+ srp | Ljubavi, Olga, hajde pođi u Fudži i čut ćeš nježnu muziku srca.
+ epo | Laŭ Ludoviko Zamenhof bongustas freŝa ĉeĥa manĝaĵo kun spicoj.
+ cze | Zvlášť zákeřný učeň s ďolíčky běží podél zóny úlů.
+ ara | أبجد هوَّز حُطّي كلَمُن سَعْفَص قُرِشَت ثَخَدٌ ضَظَغ
+ heb | עטלף אבק נס דרך מזגן שהתפוצץ כי חם
+(20 rows)
+
+DROP FOREIGN TABLE "Unicode data";
+DROP SERVER sqlite_svr;
+DROP EXTENSION sqlite_fdw;
 -- euc_jp
```

Comment:
This is `encodings` test artefacts moved to encodings.sql. Also this is applicable to other versions.

Reply:
I am sorry. This is not `encodings` test artifacts, `SELECT * FROM "Unicode data";` also was moved to this test. Thanks, @MinhLA1410 ! I haven't expect this optimization. All is ok.